articles/active-directory/develop/msal-error-handling-dotnet.md

```diff
@@ -139,6 +139,8 @@ To handle the claim challenge, you'll need to use the `.WithClaim()` method of t
 
 [!INCLUDE [Active directory error handling retries](../../../includes/active-directory-develop-error-handling-retries.md)]
 
+MsalClientException in MSAL.NET also exposes a property `IsRetryable` which is set to `true` if retry should be for the operation.
```

Comment:
@neha-bhargava This line merely restates what's [already in the API reference](https://docs.microsoft.com/en-us/dotnet/api/microsoft.identity.client.msalexception.isretryable?view=azure-dotnet#microsoft-identity-client-msalexception-isretryable) (see image below).

Instead of adding this line to this doc only restating what's already in API ref, add a `<remarks>` section to the [XML docs in the source code for the member](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/be9db130f8a9d7ec8371868f9ca71ec6b52bbdbf/src/client/Microsoft.Identity.Client/MsalException.cs#L25-L28) and explain in more detail what the developer should do.

Then, once the latest release package is published to NuGet (and the API reference is auto-updated), update this doc and add link to the API ref if you feel it's still needed.

![image](https://user-images.githubusercontent.com/13394541/157350965-54c0fe2e-cfa1-413c-8245-68f106c42cf9.png)

#please-close